src/python/WMQuality/Emulators/ReqMgrAux/MockReqMgrAux.py

```diff
@@ -1,16 +1,66 @@
+"""
+MockReqMgrAux class provides mocking methods of ReqMgrAux class
+"""
+# futures
 from __future__ import (division, print_function)
+
+# system modules
+import time
+
+# WMCore modues
 from WMCore.Agent.DefaultConfig import DEFAULT_AGENT_CONFIG
 
+
 class MockReqMgrAux(object):
     """
     Version of Services/PhEDEx intended to be used with mock or unittest.mock
     """
 
     def __init__(self, *args, **kwargs):
-        print("Using MockReqMgrAux")
+        print("Using MockReqMgrAux", args, kwargs)
+        tstamp = int(time.time())
+        cname = 'testCampaign'
+        cconf = {"go": True, "labels":["a", "b"],
+                 "overflow": {"PRIM": {}},
+                 "DDMcopies": {},
```

Comment:
This is not a WMCore campaign configuration!

Reply:
And a few other keys below don't match our campaign/transfer document schema

Reply:
And, where is campaign document schema?

Reply:
Right now in the parse script that you created: https://github.com/dmwm/WMCore/blob/master/bin/adhoc-scripts/parseUnifiedCampaigns.py#L197-L206
Until we properly define and enforce it.